src/tables.jl

```diff
@@ -38,10 +38,11 @@ function Tables.columns(A::Union{KeyedArray, NdaKa})
     L = hasnames(A) ? (dimnames(A)..., :value) :
         (ntuple(d -> Symbol(:dim_,d), ndims(A))..., :value)
     R = keys_or_axes(A)
-    G = ntuple(ndims(A)) do d
-        vec([rs[d] for rs in Iterators.product(R...)])
-        # _vec(rs[d] for rs in Iterators.product(R...))
-    end
+    R_inds = map(eachindex, R)
+    G = map(
+        (r, d) -> vec([r[rs[d]] for rs in Iterators.product(R_inds...)]),
```

Comment:
Would you mind writing this as a do-block?

Reply:
Hmmm what's the syntax for the do block for multi-arg `map`?

Reply:
```
map(A, B) do a, b
    a+b
end
```